nvim/child_process_stream.lua

```diff
@@ -15,12 +15,28 @@ function ChildProcessStream.spawn(argv)
   for i = 2, #argv do
     args[#args + 1] = argv[i]
   end
+  local env = {}
+  local i = 1
+  if argv['env'] then
+    for var, value in pairs(argv['env']) do
+      env[i] = var .. '=' .. value
+      i = i + 1
+    end
+  end
```

Comment:
Well ... i was getting a table with a dict so i just included the code in the library, I'm fine with removing it out of it.